incubator/tensorflow-inception/README.md

```diff
@@ -0,0 +1,84 @@
+# TensorFlow Serving Inception v3
+
+TensorFlow Serving is an open-source software library for serving machine learning models. This chart will specifically serve the Inception v3 model with already trained data.
+
+## TL;DR;
+
+```console
+$ helm install incubator/tensorflow-inception
+```
+
+## Introduction
+
+This chart bootstraps a TensorFlow Serving Inception v3 pod on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Get this chart
+
+Download the latest release of the chart from the [releases](../../../releases) page.
+
+Alternatively, clone the repo if you wish to use the development snapshot:
+
+```console
+$ git clone https://github.com/bitnami/tensorflow.git
```

Comment:
Should be `git clone https://github.com/bitnami/bitnami-docker-tensorflow-serving.git`